app/controllers/incomeSources/add/CheckForeignPropertyDetailsController.scala

```diff
@@ -48,7 +48,8 @@ class CheckForeignPropertyDetailsController @Inject()(val authenticate: Authenti
   extends ClientConfirmedController with FeatureSwitching with I18nSupport {
 
   def show(origin: Option[String] = None): Action[AnyContent] = Action {
-    Ok("Check Foreign Property Details - WIP")
+    Redirect(controllers.incomeSources.add.routes.ForeignPropertyAddedObligationsController.show("XAIS00000000005").url)
```

Comment:
I would assume QAs are fine with this part?

Reply:
Will check with QA, perhaps better to wait til Daniel PR merged then merge main on my branch?

Reply:
I've removed this duplicate controller now, site displays error however when hitting Ayubs page, before mine, and can't progress through journey